### PR TITLE
Fix configuration and adopt unit tests for TYPO3v10

### PR DESCRIPTION
**Pull request by tlayh**

### CODE REVIEW

**Comment by tlayh** [comment]
Die Funtional Tests laufen aufgrund des bekannten Problems nicht

File: Classes/Service/Typo3Service.php [hunk 1]
```diff
@@ -55,11 +55,11 @@ class Typo3Service implements SingletonInterface
      */
     public function __construct(Sequenzer $sequenzer)
     {
-        $this->conf = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['aoe_dbsequenzer']);
+        $this->conf = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['aoe_dbsequenzer'];
```
Comment (tomasnorre) [inline Classes/Service/Typo3Service.php]: We should use the `GeneralUtility::makeInstance(ExtensionConfiguration::class)->get('aoe_dbsequenzer')` instead, then we would have avoided a problem like this with "hardcode" path to the setting, and the serialized.

File: Tests/Unit/Service/Typo3ServiceTest.php [hunk 2]
```diff
@@ -54,7 +53,7 @@ public function setUp()
         $testConfiguration['offset'] = '1';
         $testConfiguration['system'] = 'testa';
         $testConfiguration['tables'] = 'table1,table2';
-        $GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['aoe_dbsequenzer'] = serialize($testConfiguration);
+        $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['aoe_dbsequenzer'] = $testConfiguration;
```
Comment (tomasnorre) [inline Tests/Unit/Service/Typo3ServiceTest.php]: ditto.

File: Classes/Service/Typo3Service.php (hunk 1 above)
Comment (tlayh) [reply Classes/Service/Typo3Service.php]: Changed it like recommanded and additional remove the deprecated devLog stuff

File: composer.json [hunk 3]
```diff
@@ -18,12 +18,12 @@
   "require": {
     "php": "^7.2",
     "ext-mysqli": "*",
-    "typo3/cms-core": "^8.7 || ^9.5 || ^10.4",
-    "typo3/cms-extbase": "^8.7 || ^9.5 || ^10.4",
-    "typo3/cms-sv": "^8.7 || ^9.5 || ^10.4",
-    "typo3/cms-backend": "^8.7 || ^9.5 || ^10.4",
-    "typo3/cms-reports": "^8.7 || ^9.5 || ^10.4",
-    "typo3/cms-frontend": "^8.7 || ^9.5 || ^10.4"
+    "typo3/cms-core": "^9.5 || ^10.4",
```
Comment (tomasnorre) [inline composer.json:21]: Don't forget to adjust the Test matrix in .github/workflows/Tests.yml

**Review by tomasnorre — APPROVED** [review]

**Review by TomSchenk — APPROVED** [review]

**Pull request merged**